octopusdeploy_framework/resource_community_step_template_test.go

```diff
@@ -79,6 +81,31 @@ func forceCommunityStepTemplateRefresh() {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Error forcing community step template refresh")
 	}
+
+	task := tasks.Task{}
```

Comment:
The code to actually initiate the community step template refresh was only half finished :facepalm: